docker-compose.yml

```diff
@@ -129,5 +129,30 @@ services:
                 aliases:
                     - mongo-useradm
 
+    mender-conductor:
+        image: mendersoftware/conductor:1.7.0
+        depends_on:
+            - mender-elasticsearch
+            - mender-dynomite
+        networks:
+            - mender
+
+    mender-conductor-ui:
```

Comment:
Do we need this UI for everything to work? Maybe we could put this into a separate compose file in {{extra}}